src/rpc/mining.cpp

```diff
@@ -320,9 +320,13 @@ static UniValue getmininginfo(const JSONRPCRequest& request)
         subObj.pushKV("generate", node.IsActive() && gArgs.GetBoolArg("-gen", DEFAULT_GENERATE));
         subObj.pushKV("mintedblocks", (uint64_t)node.mintedBlocks);
 
-        auto lastBlockCreationAttemptTs = node.lastBlockCreationAttemptTs;
-        subObj.pushKV("lastblockcreationattempt", FormatISO8601DateTime(node.lastBlockCreationAttemptTs));
-
+        //get the last block creation attempt by the master node
+        {
```

Comment:
We can add check here, if node does not generate we does not need to `mapMNLastBlockCreationAttemptTs` since it does not work. Can you and test for that?

Reply:
Code updated. thanks.